src/WorkflowManager/WorkflowExecuter/Common/EventMapper.cs

```diff
@@ -74,7 +74,7 @@ public static TaskUpdateEvent GenerateTaskUpdateEvent(GenerateTaskUpdateEventPar
             FailureReason failureReason,
             string message)
         {
-            Guard.Against.Null(identity, nameof(identity));
+            //Guard.Against.Null(identity, nameof(identity));
```

Comment:
We did this as we want to add this back in. This whole PR is so we can diagnose a problem in sit - we'll be adding this back in after, we just think this might be causing a hiccup in what we're trying to test.